purchases/src/main/kotlin/com/revenuecat/purchases/models/GoogleProrationMode.kt

```diff
@@ -66,6 +70,18 @@ enum class GoogleProrationMode(
     DEFERRED(BillingFlowParams.ProrationMode.DEFERRED),
     ;
 
+    /**
+     * For internal use only :)
+     */
+    val asGoogleReplacementMode: GoogleReplacementMode
```

Comment:
Is this public for testing reasons? If so we can use `@VisibleForTesting`

Reply:
Changed visibility to `internal`